frontend/src/models/models.js

```diff
@@ -86,6 +85,12 @@ module.exports = app => app.component('models', {
       }
       return 'list-default';
     },
+    getReferenceModel(schemaPath) {
+      if (schemaPath.instance !== 'ObjectId' || schemaPath.value.path == '_id') {
```

Comment:
Why is this check necessary? All you need to do is check if `ref` is set. You shouldn't need to check if the path is an ObjectId anyway, because Mongoose does support `ref` on numbers.

Reply:
Well since only objectIds can have the ref option, this check makes it so that there's no need to attempt to access a value that we know does not exist. However, in the case where it is an objectId, there is still a chance that the property does not exist which is why I use the `?.` operator to prevent it from throwing an error.